api/BaseAudioContext.json

```diff
@@ -987,12 +1025,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "53",
-              "notes": "Originally implemented on <a href='https://developer.mozilla.org/docs/Web/API/AudioContext'><code>AudioContext</code></a> in Firefox 37."
+              "version_added": "37"
```

Comment:
This matches https://developer.mozilla.org/en-US/docs/Web/API/StereoPannerNode#browser_compatibility.